build.gradle

```diff
@@ -38,6 +38,7 @@ ext.testngVersion = "6.14.3"
 ext.slf4jVersion = "1.7.30"
 ext.javaMailVersion = "1.6.2"
 ext.greenmailVersion = "1.5.11"
+//ext.jakartaActivationVersion = "1.2.2"
```

Comment:
Shall we remove this if we don't use it anymore?

Reply:
Removed.